docs/deployment.md

```diff
@@ -0,0 +1,248 @@
+platform, as it extensively uses the `edx-solutions` api. Some of the administrative tasks are also currently
+performed using proprietary Apros interface. 
+
+In order to have a working Group Project you'll need to complete three steps:
+
+1. Set up environment configuration variables
+2. Author the XBlock in Studio
+3. Configure Group Project in Apros (3rd party LMS) 
+
+## Apros configuration
+
+See corresponding document in Apros (TBD)
+
+![High level deployment overview](images/uml/high-level-architecture.png)
+
+# Setup
+
+In order to use Group Project XBlock v2 in a course, XBlock must be installed into LMS environment and enabled in 
+course's advanced settings. Also, Group Project XBlock uses a number of instance-specific configuration variables, so 
+setting them is considered a part of setup process.
+
+## Installing XBlock into LMS
+
+If you're using edx-solutions/edx-platform fork, Group Project v2 XBlock is already installed. Otherwise, installing 
+Group Project XBlock v2 is done the very same way other XBlocks are installed:
+
+1. Open terminal (or shell) to the LMS instance.
+2. Switch to LMS virtualenv
+3. pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@ **version_hash** #egg=xblock-group-project-v2
+
+*version_hash* is the hash of the git commit. By the time of writing, latest stable version was `4322ca8092c5385d5602143077e388017fcb1249`,
+so the above command looked like this:
+
+    pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@4322ca8092c5385d5602143077e388017fcb1249#egg=xblock-group-project-v2
+    
+## Setting configuration variables
+
+There are two sources of configuration variables: Django settings and XBlock-specific settings available 
+through [SettingsService][settings-service]. Both types of settings can be set via LMS environment file `lms.env.json`
+or (not recommended) directly in instance Django settings.
+
+[settings-service]: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/services.py#L7
+
+The following Django settings are used:
+
+* `EDX_API_KEY`: string - must contain a edX API Key. As this XBlock uses edX API extensively, failure to provide 
+    the key will prevent Group Project XBlock v2 from operating normally.
+* `BASE_DIR`: string - base Django instance directory. Used by Submission XBlocks as part of file storage location if
+    local file storage is used
+* `API_LOOPBACK_ADDRESS`: URL - (optional) should contain base URL of LMS API. Default: http://127.0.0.1:8000
+* File upload features piggyback on Django file storage mechanism, in order to store files, a file storage mechanism 
+    should be configured *Note:* existing production instances use S3 as file storage; using local file storage is 
+    theoretically possible, but it does not work out of the box (likely requires some modifications on LMS/Apros side).
```

Comment:
I don't understand it looks like that: (likely requires some modifications on LMS/Apros side). so both parentheses are here.